docker/release/dockerfiles/opensearch.al2.dockerfile

```diff
@@ -30,13 +30,17 @@ RUN groupadd -g $GID opensearch && \
     mkdir /tmp/opensearch
 
 # Prepare working directory
+
+ARG SECURITY_PLUGIN_DIR=$OPENSEARCH_HOME/plugins/opensearch-security
+ARG PERFORMANCE_ANALYZER_PLUGIN_DIR=$OPENSEARCH_HOME/plugins/opensearch-performance-analyzer
+
 COPY opensearch-*.tgz /tmp/opensearch/
 RUN tar -xzpf /tmp/opensearch/opensearch-`uname -p`.tgz -C $OPENSEARCH_HOME --strip-components=1 && rm -rf /tmp/opensearch && \
-    chmod 750 $OPENSEARCH_HOME/plugins/opensearch-security/tools/* && \
     mkdir -p $OPENSEARCH_HOME/data && chown -R $UID:$GID $OPENSEARCH_HOME/data
+RUN if [[ -d $SECURITY_PLUGIN_DIR ]] ; then chmod 750 $SECURITY_PLUGIN_DIR/tools/* ; fi
 COPY opensearch-docker-entrypoint.sh opensearch-onetime-setup.sh $OPENSEARCH_HOME/
 COPY log4j2.properties opensearch.yml $OPENSEARCH_HOME/config/
-COPY performance-analyzer.properties $OPENSEARCH_HOME/plugins/opensearch-performance-analyzer/pa_config/
+RUN if [[ -d $PERFORMANCE_ANALYZER_PLUGIN_DIR ]] ; then cp performance-analyzer.properties $PERFORMANCE_ANALYZER_PLUGIN_DIR/pa_config/; fi
```

Comment:
PA properties does not even exist on image, how can you simply use `cp` to solve it?

Reply:
I saw this above:

```
ARG PERFORMANCE_ANALYZER_PLUGIN_DIR=$OPENSEARCH_HOME/plugins/opensearch-performance-analyzer
```

Reply:
![Screen Shot 2021-11-29 at 1 07 51 PM](https://user-images.githubusercontent.com/65193828/143920554-15007966-1a37-4ef1-afb2-43a970b2a1bd.png)
I have just talked to @tianleh and this code wont work.
`COPY` and `cp` serve different purposes in dockerfile.

I will fix this later by COPY the entire folder then cp individual files.

Thanks.